layers/nagios.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "nagios",
   "name": "Nagios Layer",
-  "repo": "https://git.launchpad.net/nagios-layer",
+  "repo": "https://git.launchpad.net/nagios-layer/",
```

Comment:
I'm a bit confused by the addition of the `/` on the end of the line?  What's changed to need this?  (Sorry if I'm missing something obvious).  I'm just concerned it may break other charm builds?

Reply:
When I run the update readme script it changes the nagios docs link.  This seemed to be one way to prevent that but i can hunt for another solution